SoftLayer/CLI/hardware/authorize_storage.py

```diff
@@ -0,0 +1,24 @@
+"""Authorize File or Block Storage to a Hardware Server"""
+# :license: MIT, see LICENSE for more details.
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import exceptions
+from SoftLayer.CLI import helpers
+
+
+@click.command()
+@click.argument('identifier')
+@click.option('--username-storage', '-u', type=click.STRING,
+              help="The storage username to be added to the hardware server")
+@environment.pass_env
+def cli(env, identifier, username_storage):
```

Comment:
if the helpers.resolve_id is used with the storage identifier too, this would help to work with the storage id too, instead of just work with the storage user name.

```suggestion
def cli(env, identifier, storage_identifier):
    storage= SoftLayer.StorageManager(env.client)
    storage_id = helpers.resolve_id(storage.resolve_ids, storage_identifier, 'storage')
    ...
```

Reply:
I added an exception to avoid storage username error.